frontend/src/components/tables/table-view.tsx

```diff
@@ -10,6 +10,8 @@ import { useListRows } from "@/lib/hooks"
 import { DataTable } from "@/components/data-table"
 import { TableViewAction } from "@/components/tables/table-view-action"
 import { TableViewColumnMenu } from "@/components/tables/table-view-column-menu"
+import { JsonViewWithControls } from "../workbench/events/events-selected-action"
```

Comment:
Could we:

1. Move `JsonViewWithControls` into a separate file under `/components/json-viewer.tsx`
2. Import it in thie file and in `/workbench/events/events-selected-action`  using its absolute path prefixed with @ i.e. `@/components/json-viewer.tsx`